### ISSUE TITLE
Developer documentation

### ISSUE THREAD

**Issue by jachym** [geopython/pywps#281]
- [ ] Class diagram
- [ ] Class description

**Comment by tomkralidis** [comment]
Can we use Sphinx apidoc functionality here?

**Comment by tomkralidis** [comment]
It seems we already have this? See http://pywps.readthedocs.io/en/latest/api.html

**Comment by ldesousa** [comment]
Yes, Jachym started it up when we were in Como, I believe. You can follow on with it if you will. My task at this stage is to produce the Domain Model(s). We then need more detailed class descriptions, that's where Sphinx may help.
